state/state.go

```diff
@@ -538,6 +540,8 @@ func (s *State) Stage(newVer trie.Version) (*Stage, error) {
 					return err
 				}
 			}
+			// Just once for the account trie.
```

Comment:
I would suggest avoid including metrics in state, as it's the primitive operation and all the metrics will be fallen into trie node cache.